src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -351,6 +354,11 @@ export default {
 				this.trackLoans();
 			}
 		});
+
+		// Verify challenge header and show team picks
+		if (this.showChallengeHeader) {
+			this.showTeamPicks = true;
```

Comment:
Can we just use `showChallengeHeader` and remove `showTeamPicks`?

Reply:
This is the initial state for showTeamPicks param if we are showing the header.
About your comment above, I understand that we can remove recommended loans with the team picks switch; if not, should we block the switch to be always on if is the page is under a challenge?